### PR TITLE
chore(Changelog): Press Esc to cancel selected text

### PR DESCRIPTION
**Pull request by coder3101**
## Checklist

- [ ] If the key of a setting is changed, the `old` attribute is updated or it is resolved in SettingsUpdater.
- [ ] If there are changes of the text displayed in the UI, they are wrapped in `tr()` or `QCoreApplication::translate()`.
- [ ] If needed, I have opened a pull request or an issue to update the [documentation](https://github.com/cpeditor/cpeditor.github.io).
- [ ] If these changes are notable, they are documented in [CHANGELOG.md](https://github.com/cpeditor/cpeditor/blob/master/CHANGELOG.md).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 -   Configurable text cursor width. (#781 and #795)
 -   Now accepted testcases can be auto-unchecked. (#734 and #797)
 -   Update icon for macOS. (#838)
+-   Press Esc key to cance selected text. (#786)
```
Comment (coder3101) [inline CHANGELOG.md:9]:
```suggestion
-   Press Esc key to cancel selected text. (#786)
```

**Review by sonulohani — COMMENTED** [review]
The changes for esc key to cancel selection was in another repositories. Do we still add the changes in changelog here. I think the appropriate changes was to show one time dialog when F11 is pressed.

**Comment by coder3101** [comment]
> The changes for esc key to cancel selection was in another repositories. Do we still add the changes in changelog here. I think the appropriate changes was to show one time dialog when F11 is pressed.

The issue for that request was raised on this repository. So we have to add to change log so users know that it has been fixed.

**Comment by coder3101** [comment]
This was better suited for `Changed` section of Changelog.

**Review by sonulohani — DISMISSED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,10 @@
 -   Fix that the Find/Replace dialog is not floating in i3-wm. (#767)
 -   Fix text selection color when Dark Fusion is selected. (#788)
 
+## Changed
+
+-   Esc key now cancels selected text, F11 can be used to enter/exit full screen mode. (#786)
```
Comment (ouuan) [inline CHANGELOG.md:17]:
```suggestion
-   Esc key, which was used to exit full-screen mode, now cancels text selection. Now you can use F11 to enter/exit the full-screen mode. (#786)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,6 +12,10 @@
 -   Fix that the Find/Replace dialog is not floating in i3-wm. (#767)
 -   Fix text selection color when Dark Fusion is selected. (#788)
```
Comment (ouuan) [inline CHANGELOG.md:13]:
```suggestion
-   Fix text selection color when Dark Fusion is selected. (#788)
-   Now entering/exiting the full-screen mode outside of CP Editor (via the OS/DE/WM) is correctly handled. (#833)
```

**Review by ouuan — CHANGES_REQUESTED** [review]

**Review by ouuan — APPROVED** [review]

**Review by sonulohani — APPROVED** [review]

**Comment by ouuan** [comment]
@allcontributors add @sonulohani for review

**Pull request merged**